packages/components/src/Menu/Menu.tsx

```diff
@@ -12,31 +10,34 @@ import classnames from "classnames";
 import { AnimatePresence, motion } from "framer-motion";
 import { useOnKeyDown } from "@jobber/hooks/useOnKeyDown";
 import { useRefocusOnActivator } from "@jobber/hooks/useRefocusOnActivator";
+import { useWindowDimensions } from "@jobber/hooks/useWindowDimensions";
 import { IconNames } from "@jobber/design";
-import { useSafeLayoutEffect } from "@jobber/hooks/useSafeLayoutEffect";
+import { usePopper } from "react-popper";
+import { useIsMounted } from "@jobber/hooks/useIsMounted";
+import ReactDOM from "react-dom";
+import { useFocusTrap } from "@jobber/hooks/useFocusTrap";
 import styles from "./Menu.css";
 import { Button } from "../Button";
 import { Typography } from "../Typography";
 import { Icon } from "../Icon";
+import { formFieldFocusAttribute } from "../FormField/hooks/useFormFieldFocus";
+
+const SMALL_SCREEN_BREAKPOINT = 489;
+const MENU_OFFSET = 6;
 
 const variation = {
   overlayStartStop: { opacity: 0 },
-  startOrStop: (position: string) => {
+  startOrStop: (placement: string | undefined) => {
     let y = 10;
 
-    if (position === "below") y *= -1;
+    if (placement?.indexOf("bottom") !== -1) y *= -1;
```

Comment:
This is very much a personal style, both achieve the same goal (so feel free to ignore), but `includes` instead of `indexOf` is a bit tidier because it returns a boolean vs having to check for -1 with indexOf.

Reply:
that's fair, I'd agree that's a nicer check. I'll update that.